index.md

```diff
@@ -14,31 +14,20 @@ Note: the nested block doesn't doesn't include function or class definition
 as a block, so the example below is considered to have *four* nested blocks,
 not five.
 
-```python
-def my_f(num):
-    if num > 0:
-        if num > 3:
-            if num > 10:
-                if num < 50:
-                    print(num)
-    else:
-        print('done')
-```
```

Comment:
note: of course, the downside of using all these includes instead of the codeblocks, is that we cannot see the markdown rendered with the code on GitHub...